packages/manager/apps/dedicated/client/app/billing/payment/method/routing.js

```diff
@@ -66,6 +80,73 @@ export default /* @ngInject */ (
           });
         }
       },
+      isSplitPaymentAvailable: /* @ngInject */ (
+        $http,
+        currentUser,
+        ovhFeatureFlipping,
+        splitPaymentTag,
+      ) =>
+        ovhFeatureFlipping
+          .checkFeatureAvailability(SPLIT_PAYMENT_FEATURE_NAME)
+          .then((feature) =>
+            feature.isFeatureAvailable(SPLIT_PAYMENT_FEATURE_NAME),
+          )
+          .then((isFeatureAvailable) =>
+            isFeatureAvailable
+              ? $http
+                  .get('/me/tag/available')
+                  .then(({ data: tags }) =>
+                    tags.some(
+                      ({ name: tagName }) => tagName === splitPaymentTag,
+                    ),
+                  )
+              : false,
+          ),
+      splitPayment: /* @ngInject */ (
+        $http,
+        isSplitPaymentAvailable,
+        splitPaymentTag,
+        tagStatusEnum,
+      ) =>
+        isSplitPaymentAvailable
+          ? $http
+              .get(`/me/tag/${splitPaymentTag}`)
+              .then(({ data }) => data)
+              .then((tagInfo) => ({
+                canBeActivated: [
+                  tagStatusEnum.DELETED,
+                  tagStatusEnum.REFUSED,
+                ].includes(tagInfo?.status),
+                canBeDeactivated: tagStatusEnum.CREATED === tagInfo.status,
+                ...tagInfo,
+              }))
+              .catch(() => ({
+                canBeActivated: true,
+                canBeDeactivated: false,
+              }))
+          : null,
+      splitPaymentTag: /* @ngInject */ () =>
+        // currentUser // TODO: will be set once ok is ok
+        SPLIT_PAYMENT.IT, // TODO: remove code once test is ok
+      // SPLIT_PAYMENT[currentUser.ovhSubsidiary],
+      tagStatusEnum: /* @ngInject */ ($http, isSplitPaymentAvailable) =>
+        isSplitPaymentAvailable
+          ? $http
+              .get('/me.json')
+              .then(({ data: schema }) => schema)
+              .then((schema) =>
+                schema.models['me.tag.StatusEnum'].enum.reduce(
+                  (list, status) => ({
+                    ...list,
+                    [status]: status,
+                  }),
+                  {},
+                ),
+              )
+          : [],
+      splitPaymentInformationHref: /* @ngInject */ (currentUser, CORE_URLS) =>
+        CORE_URLS.splitPaymentInformation.IT, // TODO: remove code once test is ok
```

Comment:
Should be removed ?

Reply:
Yes in fact it was to allow test without an italian nichandle. But as QA team will provide me one, it will be removed now